docs_versioned_docs/version-ros2jazzy/ros/config/yaml/platform/controller.mdx

```diff
@@ -17,3 +17,53 @@ Switching between each is done by setting the `controller` parameter under the `
 ```yaml
 controller: ps4 # one of ps4, ps4, logitech, xbox
 ```
+
+## Teleop Speeds
+
+By default, holding the left trigger button of the controller enables driving at "normal" speeds
+while holding the right trigger button enables driving at "turbo" speeds.
+
+The table below shows the default linear and angular speeds for each platform:
+
+| Platform   | Linear speed (normal, m/s) | Linear speed (turbo, m/s) | Angular speed (normal, rad/s) | Angular speed (turbo, rad/s) |
```

Comment:
This is good data for the docs website, to simplify review for a customer.
Is this something Platform Software is willing to maintain, even as the standard yaml file gets updated?